### PR TITLE
README - add notes about using Blazegraph on Ubuntu

### LINKED ISSUE #28: Update README for blazegraph
References to `bigdata` might need to be replaced with `blazegraph`, e.g.
- http://localhost:9999/bigdata/#update -> http://localhost:9999/blazegraph/#update

### PR DESCRIPTION
**Pull request by dazza-codes**
Fixes #28 (for Ubuntu at least, not sure about OSX instructions)

### CODE REVIEW

**Comment by gkellogg** [comment]
Question on these PRs is who is doing the pull? I had some long outstanding PRs that I was waiting for @cbeer to approve, but it dropped off the radar. Project ownership definitely needs to be resolved.

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Tests can be run using `rspec`
 
 Service can be run locally using `bundle exec rails server`
 
-### Running on Blazegraph
+### Running on Blazegraph - for OSX
```
Comment (gkellogg) [inline README.md]: MacOS now, I believe.

Comment (dazza-codes) [reply README.md]: OK, changed to `macOS` after looking at the Apple-Logo -> About this Mac on my laptop running macOS Sierra.

**Comment by dazza-codes** [comment]
Anyone with permissions on this repo can merge a PR (AFAIK).  FYI, a small group at DLSS are briefly looking at this to explore whether we can use it for discovery on a Bibframe2 KB of catalog records.

**Comment by gkellogg** [comment]
All my changes I had put on @cbeer, as they have general BlackLight architectural implications, as well as standard gems, so I don't feel qualified to do those PRs.

Things such as this, i can certainly handle, and will do so.

**Pull request merged**